docs/examples/CementiteAnalysis.rst

```diff
@@ -10,7 +10,7 @@ http://dx.doi.org/10.1016/j.calphad.2010.01.004.
 (http://www.sciencedirect.com/science/article/pii/S0364591610000052)
 
 The TDB file used here differs slightly from the published TDB to ensure
-compatibility with pycalphad's TDB parser. All phases except cementite
```

Comment:
This change with commas will break formatting?

Reply:
I think it's just a pandoc/unicode thing. Again, looks like it produces the same character to me when I generated the docs myself